src-tauri/Cargo.toml

```
@@ -1,10 +1,10 @@
 [package]
 name = "weather"
-version = "0.0.0"
-description = "A Tauri App"
-authors = ["you"]
-license = ""
-repository = ""
+version = "1.0.0"
+description = "A Weather App"
+authors = ["Daniel", "Kenny", "Max"]
```

We probably should have this with email (so git format I think?), as they suggest [here](https://doc.rust-lang.org/cargo/reference/manifest.html#the-authors-field). 
So that would be something like this:

```suggestion
authors = ["Daniel Agyapong <kunkumah@gmail.com>", "Kenny Ketelaar <kenny@ketelaar.tech>", "Max Meijer <mdvmeijer@protonmail.com>"]
```

---

Yeah I think that works too. We can go ahead with that.